concordium-std-derive/src/lib.rs

```diff
@@ -1713,3 +1713,24 @@ pub fn concordium_cfg_test(_attr: TokenStream, item: TokenStream) -> TokenStream
     };
     out.into()
 }
+
+/// Sets the cfg exclusively for building, that will be disabled for Wasm and
+/// native tests. Opposite to `#[concordium_cfg_test]` counterpart.
```

Comment:
```suggestion
/// If `wasm-test` feature of `concordium-std` is enabled ignore the item. Otherwise this is equivalent to `#[cfg(not(test))]`. Use as a dual to `#[concordium_cfg_test]`.
```

Reply:
Not really a fan of the proposed wording. User is usually not aware of `wasm-test` feature and when it gets enabled. I decided that it may be better to elaborate when this feature gets enabled automatically.